mne/viz/tests/test_topo.py

```diff
@@ -77,6 +77,11 @@ def test_plot_topo():
     # Show topography
     evoked = _get_epochs().average()
     plot_evoked_topo(evoked)  # should auto-find layout
+    # Test jointplot
+    evoked.plot_joint()
+    evoked.plot_joint(title='test', ts_args=dict(spatial_colors=True),
+                      topomap_args=dict(colorbar=True, times=[0.]))
```

Comment:
Then we're returning 4 figures in that function ... a lot IMO.
I'd prefer to use an example with only one channel type. I want to make an example for linear_regression_raw with continuous covariates anyways because it's not quite clear to people how it works for now, I just need to make a file for that available.

But I guess for now it'd be best to use plot_evoked_erf_erp.